projects/laji/src/app/shared-modules/select/metadata-select/metadata-select.component.ts

```diff
@@ -134,6 +135,7 @@ export class MetadataSelectComponent implements OnChanges, OnDestroy, ControlVal
       ))
     ).subscribe(options => {
         this.setOptions(options);
+        this._options = this.deleteOptionsfromExcludedOptions(this._options, this.excludeOptions);
```

Comment:
this could be moved to the pipe so that options here would be already correct. For example just above line 132 (then using `labelAsValue` wouldn't break this behavior). Somethinkg like:
 ```typescript
 map(options => this.deleteOptionsfromExcludedOptions(options, this.excludeOptions))

Reply:
Agree